stream-chat-android-ui-components/src/main/res/drawable-night/stream_ui_divider.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <size android:height="1dp" />
+</shape>
```

Comment:
What colour will this end up being?

Reply:
It will be a divider with no colour. In the "Add Group Members" screen I don't see any divider. Take a look: figma.com/file/ekifwChR9tR7zRJg1QEzSM/Q2_Stream_Mobile_Chat_v1?node-id=0%3A1439

Reply:
Let's ask about this, seems to be a mistake in the design rather than intentional, all of the neighbouring screens have dividers

Reply:
Added the divider than. This file got deleted.